components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```diff
@@ -356,6 +353,18 @@ def run_clp(
                         last_archive_stats,
                     )
                     db_conn.commit()
+                if StorageEngine.CLP_S == clp_storage_engine:
+                    indexer_cmd = [
```

Comment:
```suggestion
                    # TODO: Since CLP doesn't currently support datasets but users of the index
                    # require a dataset name, we hardcode a name for now.
                    dataset_name = "default"
                    indexer_cmd = [
```